### PR TITLE
Feature/27 redesign with adverity design

### LINKED ISSUE #27: Redesign Application with Adverity Design System
- [x] redesign `Rule` component 
-> replace `input` (for `match`, `tag` and `tagType`)
-> replace `Select` (for `tagType`)
-> replace `Checkbox` (for `isRegExp`)
- [x] redesign `RuleSet` component 
-> replace `Select` (for `sourceDimension`)
-> replace `Button` and use Adverity icons (for adding `rule` and deleting `rule`)
- [x] redesign `TagRules` component 
-> replace `Button` and use Adverity icons (for adding `rule` and deleting `rule`)

### PR DESCRIPTION
**Pull request by jasrial**
closes #27

### CODE REVIEW

File: src/components/Rule.js [hunk 1]
```diff
@@ -72,30 +73,33 @@ const Rule = ({ rule, dimensions, ruleSetId }: Props) => {
...
-              placeholder="Choose dimension..."
-              options={((dimensions: any): OptionsType)}
-              value={
-                isNil(findDimension(dimensions, tagType))
-                  ? null
-                  : findDimension(dimensions, tagType)
-              }
-              isClearable
-              onChange={selectedDimension =>
+            <Input
```
Comment (vlabs) [inline src/components/Rule.js]: please create issue to use `<Select/>` once design-system is ready

https://design-system.adverity.net/1.0.0-alpha.6/storybook/?path=/docs/components-pattern-select--simple

Comment (jasrial) [reply src/components/Rule.js]: https://github.com/jasrial/tag-rules/issues/30

File: src/App.js [hunk 2]
```diff
@@ -17,11 +18,13 @@ const App = () => {
 
   return (
     <DispatchContext.Provider value={dispatch}>
-      <div className="App">
-        <br />
-        <TagRules ruleSets={state} dimensions={dimensions} />
-        <br />
-      </div>
+      <ThemeProvider theme={defaultTheme}>
+        <div className="App">
```
[the remark this answers is not in the document]
Comment (jasrial) [reply src/App.js]: 5111e629acec8c121eb0d377de00805db23a2180

File: src/App.js [hunk 3]
```diff
@@ -17,11 +18,13 @@ const App = () => {
...
+          <br />
```
[the remark this answers is not in the document]
Comment (jasrial) [reply src/App.js]: 53d097c2c1534681b5bab56ff5f6d831982d344e

File: src/components/Rule.js [hunk 4]
```diff
@@ -38,16 +36,19 @@ const Rule = ({ rule, dimensions, ruleSetId }: Props) => {
             </p>
           </Col>
           <Col md={4}>
-            <FormControl
+            <Input
               type="text"
```
[the remark this answers is not in the document]
Comment (jasrial) [reply src/components/Rule.js]: 001700432f597fa40563ddc7a2337945869a7a35

File: src/components/Rule.js [hunk 5]
```diff
@@ -38,16 +36,19 @@ const Rule = ({ rule, dimensions, ruleSetId }: Props) => {
...
               onChange={event => dispatch(setMatchForm(event, rule, ruleSetId))}
             />
           </Col>
           <Col md={4}>
-            <Form.Check
+            <Input
               type="checkbox"
               label="RegExp?"
+              size="xs"
+              mr="xs"
```
[the remark this answers is not in the document]
Comment (jasrial) [reply src/components/Rule.js]: 10929e6a6f56080513a9ccaf091c3cd75c91ff58

**Pull request merged**